client/server/pages/index.js

```diff
@@ -830,12 +830,16 @@ function wpcomPages( app ) {
 	} );
 
 	app.get( [ '/subscriptions', '/subscriptions/*' ], function ( req, res, next ) {
-		if ( req.context.isLoggedIn ) {
+		const showSitesViewToInternalUsers =
+			config.isEnabled( 'subscription-management/internal-users/sites-view' ) &&
+			req.path === '/subscriptions/sites';
```

Comment:
We do not have to do the: 
```
req.path === '/subscriptions/sites'
```
As long as we ensure that the tabs within the subscriptions' portal point the logged-in user back to the re-skinned portal when the Comments, the Pending, or the Settings tab is clicked.

Reply:
In fact, if we do not do the check, then it will make it easier for us to test the other views before we have a feature flag for them. 😁

Reply:
True. Removing the check.